dp_creator_ii/__init__.py

```diff
@@ -29,13 +37,12 @@ def get_arg_parser():
 
 def main():  # pragma: no cover
     # We call parse_args() again inside the app.
-    # We only call it here so "--help" is handled.
+    # We only call it here so "--help" is handled,
+    # and to validate inputs.
     get_arg_parser().parse_args()
```

Comment:
Can we also have a test that the required arguments were passed?  When I run the app with no args, it shows a stack trace of an unhandled error. I read your comment about skipping this for now, but I'm not sure I understand how that helps with debugging?